templates/master.yaml

```diff
@@ -729,22 +617,36 @@ coreos:
 
       [Install]
       WantedBy=multi-user.target
+  - name: k8s-key-generator.service
+    enable: true
+    command: start
+    content: |
+      [Unit]
+      Description=k8s-key-generator Service
+
+      [Service]
+      Type=oneshot
+      ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/secrets
+      ExecStart=/usr/bin/openssl genrsa -out /etc/kubernetes/secrets/token_sign_key.pem 2048
+
+      [Install]
+      WantedBy=multi-user.target
```

Comment:
Those keys are [already present](https://github.com/giantswarm/k8scloudconfig/blob/58aa6022a5209fecfdb0b6e85f1e2af50a080cd2/templates/master.yaml#L678) on the machines in the KVM setup I suppose. We need to generate them.